api/app/utils/daily_pdf_gen.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Mapping
+import pdfkit
+from jinja2 import Template
+
+from app.schemas.hfi_calc import FireCentre, HFIResultResponse, PlanningArea, WeatherStation
+from app.utils.pdf_data_formatter import response_2_daily_jinja_format
+
+daily_template_path = "api/app/utils/daily_template.html"
+daily_rendered_path = "api/app/utils/daily_rendered.html"
```

Comment:
You won't have write access here. You'll need to store it in a temporary file - it also won't work with multiple workers/threads. Ideally in memory.